lib/cli/commands/getinfo.ts

```diff
@@ -29,12 +29,38 @@ ${info.urisList[0].substring(info.urisList[0].indexOf('@'))}` : '';
   console.log(basicInfotable.toString(), '\n');
 };
 
+const determineXudStatus = (lndMap: [string, LndInfo.AsObject][], raiden: RaidenInfo.AsObject | undefined) => {
+  let status = '';
+
+  /*const peers = this.listPeers();
```

Comment:
Why call `this.listPeers()` if we already have the number of peers in the response of `GetInfo` as `info.numPeers`?

Reply:
Probably because I wrote it that way in the issue...

Agree - just check if this number is >0 does the trick!